spec/requests/api/v1/collections_spec.rb

```diff
@@ -199,132 +198,183 @@
+          description: 'description 1 common content',
+          content: 'content 1 common content',
+          created: datetime,
+          path: 'http://www.agency.gov/page1.html',
+          promote: true, updated: datetime,
+          updated_at: datetime }
+      end
+      let(:hash2) do
+        { _id: 'a2',
+          language: 'en',
+          title: 'title 2 common content',
+          description: 'description 2 common content',
+          content: 'other unrelated stuff',
+          created: datetime.to_s,
+          path: 'http://www.agency.gov/page2.html',
+          promote: false, tags: 'tag1, tag2',
+          updated_at: datetime }
       end
-    end
 
-    context 'no results' do
-      before do
-        clear_index(collections_index_name)
-        post '/api/v1/collections', params: valid_params, headers: valid_session
-        clear_index(documents_index_name)
+      describe 'status' do
+        subject { response.status }
+
+        it { is_expected.to eq(200) }
+      end
+
+      describe 'body' do
+        let(:body) { JSON.parse(response.body) }
+        let(:result1) do
+          {
+            'language' => 'en',
+            'created' => datetime,
+            'path' => 'http://www.agency.gov/page1.html',
+            'title' => 'title 1 common content',
+            'description' => 'description 1 common content',
+            'content' => 'content 1 common content',
+            'changed' => datetime
+          }
+        end
+        let(:result2) do
+          {
+            'language' => 'en',
+            'created' => datetime,
+            'path' => 'http://www.agency.gov/page2.html',
+            'title' => 'title 2 common content',
+            'description' => 'description 2 common content',
+            'changed' => datetime
+          }
+        end
+
+        it 'returns highlighted JSON search results' do
+          expect(body).to match(hash_including('results' => [result1, result2]))
+        end
+
+        describe 'metadata' do
+          let(:metadata) { body['metadata'] }
+          let(:suggestion_hash) do
+            { 'text' => 'common content',
+              'highlighted' => 'common content' }
+          end
+
+          it 'returns highlighted JSON suggestion' do
+            expect(metadata['suggestion']).to match(hash_including(suggestion_hash))
+          end
+
+          it 'returns a non-zero results total' do
+            expect(metadata['total']).to be > 0
+          end
+
+          context 'when aggregation field is not present in the documents' do
+            it 'does not return an aggregation for that field' do
+              expect(metadata['aggregations'].pluck('audience')).to eq([nil])
+            end
+          end
+
+          context 'when aggregation field is present in the documents' do
+            it 'returns aggregations for that field' do
+              expect(metadata['aggregations'].pluck('tags')).not_to eq([nil])
```

Comment:
This seems to be the only spec testing that any of the aggregations return any sort of data. Can you add some specs that will test that the expected aggregation data is returned for a search that returns results?